services/name-gender-bio.js

```diff
@@ -29,24 +28,24 @@ module.exports.addGender = (id, gender) => {
 };
 
 module.exports.addName = async (id, name) => {
-  console.log("in add name")
+  console.log("in add name");
   try {
     const profile = await Profile.find({ userId: id });
-  if (profile) {
-    console.log("in update")
-     const res = await Profile.updateOne(
-      { userId: id },
-      {
-        $set: {
-          name: name,
-        },
-        $currentDate: { lastUpdated: true },
-      }
-    );
-//   return res
-  }
+    if (profile) {
+      console.log("in update");
+      const res = await Profile.updateOne(
+        { userId: id },
+        {
+          $set: {
+            name: name,
+          },
+          $currentDate: { lastUpdated: true },
+        }
+      );
+      //   return res
```

Comment:
remove this line

Reply:
Maybe we can delete this whole file? and use the update endpoint in profile.js instead

Reply:
let's change it. this route is called addName -  but looking at the code, it looks like it is updating the name? so this route is for adding the name for the very first time?
We can't use update function for all of them, because some are adding the value, not updating it, but we need to separate them so we know where we do POST name because there is no name, and where we update the existing name. 
Add should be used where we add a value, update - where we actually update. POST, PUT/PATCH
I think the logic is also that we create an empty profile first when we try to get one -  and then we start adding (in fact updating) fields. Since we don't need to call API on each page, it prob makes sense to not create a profile in the beginning? and only create one at the end -  so logic of getProfile should also be changed